data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1559,12 +1559,47 @@
     "book_learn": [ [ "textbook_extraction", 4 ] ],
     "qualities": [ { "id": "CONTAIN", "level": 1 } ],
     "tools": [ [ [ "syringe", -1 ] ], [ [ "surface_heat", 1, "LIST" ] ] ],
-    "components": [
-      [ [ "hi_q_distillate", 10 ] ],
-      [ [ "element", 1 ], [ "crude_heating_element", 1 ] ],
-      [ [ "plastic_chunk", 1 ] ],
-      [ [ "glass_shard", 1 ] ]
-    ]
+    "components": [ [ [ "hi_q_distillate", 10 ] ], [ [ "element", 1 ] ], [ [ "plastic_chunk", 1 ] ], [ [ "glass_shard", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "formic_acid",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_DRUGS",
+    "skill_used": "chemistry",
+    "difficulty": 2,
+    "time": "20 m",
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "autolearn": true,
+    "//": "Should likely come from an entomology text, or something similar, in the future; but still relatively basic.",
+    "tools": [ [ [ "mortar_pestle", -1 ], [ "funnel", -1 ] ] ],
+    "components": [ [ [ "wasp_sting", 1 ], [ "bee_sting", 1 ] ] ]
+  },
+  {
+  	"type": "recipe",
+  	"activity_level": "LIGHT_EXERCISE",
+  	"result": "latex",
+  	"category": "CC_CHEM",
+  	"subcategory": "CSC_CHEM_OTHER",
+  	"skill_used": "chemistry",
+  	"difficulty": 2,
+  	"time": "3 m",
+  	"qualities": [ { "id": "CONTAIN", "level": 1 } ],
+  	"tools": [ [ [ "mortar_pestle", -1 ] ] ],
+  	"components": [ [ [ "spurge", 1 ] ] ]
+  },
+  {
+  	"type": "recipe",
+  	"activity_level": "LIGHT_EXERCISE",
+  	"result": "chunk_rubber",
+  	"category": "CC_CHEM",
+  	"subcategory": "CSC_CHEM_OTHER",
+  	"skill_used": "chemistry",
+  	"difficulty": 3,
+  	"time": "30 m",
+  	"//": "This is not actually accurate; producing rubber from formic acid and latex generally takes around 12 hours. However, it will be better if we simply create some kind of curing rubber, and let it naturally evolve into rubber chunks from there. Nobody has twelve hours to watch rubber form in a zombie apocalypse.",
+  	"components": [ [ [ "latex", 5 ], ["formic_acid", 2 ] ] ]
```

Comment:
```suggestion
    "components": [ [ [ "hi_q_distillate", 10 ] ], [ [ "element", 1 ] ], [ [ "plastic_chunk", 1 ] ], [ [ "glass_shard", 1 ] ] ]
  },
  {
    "type": "recipe",
    "activity_level": "LIGHT_EXERCISE",
    "result": "formic_acid",
    "category": "CC_CHEM",
    "subcategory": "CSC_CHEM_DRUGS",
    "skill_used": "chemistry",
    "difficulty": 2,
    "time": "20 m",
    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
    "autolearn": true,
    "//": "Should likely come from an entomology text, or something similar, in the future; but still relatively basic.",
    "tools": [ [ [ "mortar_pestle", -1 ], [ "funnel", -1 ] ] ],
    "components": [ [ [ "wasp_sting", 1 ], [ "bee_sting", 1 ] ] ]
  },
  {
    "type": "recipe",
    "activity_level": "LIGHT_EXERCISE",
    "result": "latex",
    "category": "CC_CHEM",
    "subcategory": "CSC_CHEM_OTHER",
    "skill_used": "chemistry",
    "difficulty": 2,
    "time": "3 m",
    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
    "tools": [ [ [ "mortar_pestle", -1 ] ] ],
    "components": [ [ [ "spurge", 1 ] ] ]
  },
  {
    "type": "recipe",
    "activity_level": "LIGHT_EXERCISE",
    "result": "chunk_rubber",
    "category": "CC_CHEM",
    "subcategory": "CSC_CHEM_OTHER",
    "skill_used": "chemistry",
    "difficulty": 3,
    "time": "30 m",
    "//": "This is not actually accurate; producing rubber from formic acid and latex generally takes around 12 hours. However, it will be better if we simply create some kind of curing rubber, and let it naturally evolve into rubber chunks from there. Nobody has twelve hours to watch rubber form in a zombie apocalypse.",
    "components": [ [ [ "latex", 5 ], [ "formic_acid", 2 ] ] ]
```